src/ensembl/src/content/app/browser/browser-region-editor/BrowserRegionEditor.test.tsx

```diff
@@ -34,7 +29,7 @@ describe('<BrowserRegionEditor', () => {
     chrLocation: initialChrLocation,
     genomeKaryotype: createGenomeKaryotype(),
     isActive: true,
-    isDisabled: false,
+    shouldBeOpaque: false,
```

Comment:
As above: this prop name is too specific, tightly coupled with the way we represent the disabled state of this component, and not very informative for the consumer of the component.

Reply:
✅